boards/arm/npcx9m6f_evb/doc/index.rst

```diff
@@ -94,10 +94,32 @@ JTAG-only sessions.
 Flashing
 ========
 
-Build application as usual for the ``npcx9m6f_evb`` board, and flash
-using Servo V2, μServo, or Servo V4 (CCD). See the
+If the correct headers are installed, this board supports both J-TAG and also
+the ChromiumOS servo.
+
+To flash using Servo V2, μServo, or Servo V4 (CCD), see the
 `Chromium EC Flashing Documentation`_ for more information.
 
+To flash with J-TAG, install the drivers for your programmer, for example:
+SEGGER J-link's drivers are at https://www.segger.com/downloads/jlink/
+
+The openocd from Zephyr SDK 0.14.2 doesn't include npcx support, so build openocd from source.::
```

Comment:
Drop period from end of line, since you already have the colon

Reply:
The double colon doesn't appear in the generated HTML, :: makes the next section a <pre> block.